### PR TITLE
Add build to docker daemon to changelog

### PR DESCRIPTION
**Pull request by TadCordle**

### CODE REVIEW

File: jib-maven-plugin/CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ All notable changes to this project will be documented in this file.
 - Better feedback for build failures ([#197](https://github.com/google/jib/pull/197))
 - Warns if specified `mainClass` is not a valid Java class ([#206](https://github.com/google/jib/issues/206))
 - Warns if build may not be reproducible ([#245](https://github.com/GoogleContainerTools/jib/pull/245))
+- `jibBuildDocker` gradle task and `jib:buildDocker` maven goal to build straight to Docker daemon ([#243](https://github.com/GoogleContainerTools/jib/issues/243))
```
Comment (coollog) [inline jib-maven-plugin/CHANGELOG.md]: Oh this should be split up between maven and gradle

**Comment by coollog** [comment]
Could also move the proposal to archives in this PR

**Comment by TadCordle** [comment]
What about updating the READMEs? Is that later?

**Comment by coollog** [comment]
Yeah READMEs will be updated after the new version is released

File: jib-maven-plugin/CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ All notable changes to this project will be documented in this file.
 - Better feedback for build failures ([#197](https://github.com/google/jib/pull/197))
 - Warns if specified `mainClass` is not a valid Java class ([#206](https://github.com/google/jib/issues/206))
 - Warns if build may not be reproducible ([#245](https://github.com/GoogleContainerTools/jib/pull/245))
+- `jibBuildDocker` gradle task to build straight to Docker daemon ([#265](https://github.com/GoogleContainerTools/jib/pull/265))
```
Comment (coollog) [inline jib-maven-plugin/CHANGELOG.md]: Oh I meant move this to `jib-gradle-plugin/CHANGELOG.md`.

File: jib-maven-plugin/CHANGELOG.md [hunk 3]
```diff
@@ -13,6 +13,8 @@ All notable changes to this project will be documented in this file.
...
+- `jib:buildDocker` maven goal to build stranght to Docker daemon ([#266](https://github.com/GoogleContainerTools/jib/pull/266))
```
Comment (coollog) [inline jib-maven-plugin/CHANGELOG.md]: typo on stranght

File: jib-maven-plugin/CHANGELOG.md (hunk 2 above)
Comment (TadCordle) [reply jib-maven-plugin/CHANGELOG.md]: Oh, should it go in both? i.e. base changelog mentions both, and then each plugin changelog mentions the respective task/goal? Or just the plugin changelogs

Comment (TadCordle) [reply jib-maven-plugin/CHANGELOG.md]: Wait I see what's happening, nevermind I know what to do

**Pull request merged**